angular-spa-ui/js/common/modal/index.html

```diff
@@ -0,0 +1,17 @@
+<script type="text/ng-template" id="myModalContent.html">
+        <div class="modal-header">
```

Comment:
Setup codestyle in your IDE... 1 tab symbol should be replaced with 4 spaces. I see 8.